source/Server/AzureDevOpsIssueTracker.cs

```diff
@@ -19,6 +19,6 @@ public AzureDevOpsIssueTracker(IAzureDevOpsConfigurationStore configurationStore
 
         public bool IsEnabled => configurationStore.GetIsEnabled();
 
-        public string? BaseUrl => configurationStore.GetIsEnabled() ? configurationStore.GetBaseUrl() : null;
+        public string? BaseUrl => null;
```

Comment:
This is a bit confusing, this is not really being used in Server, so it is safe to set it to null